list.js

```diff
@@ -0,0 +1,106 @@
+function chceckIdCategory() {
+  var splitOne = $(location).attr('href').split('#');
+  if (splitOne[1] != null) splitOne = splitOne[1].split('%')[0];
+  else splitOne = -1;
+
+  return splitOne;
+}
+
+function makingStars(val) {
+  $('.stars').barrating({
+    theme: 'fontawesome-stars-o',
+    initialRating: val,
+    hoverState: false,
+    readonly: true,
+  });
+}
+
+function showCategory() {
+  if (id != -1)
+    var localsToShow = locals
+      .filter(l => {
+        return l.category_id === id;
+      })
+      .sort((a, b) => {
+        return b.rate - a.rate;
+      });
+  else
+    var localsToShow = locals.sort((a, b) => {
+      return b.rate - a.rate;
+    });
+
+  if (localsToShow.length == 0) $('#list').html('Brak lokali w tej kategorii');
+  else {
+    $('#list').html(
+      '<table class="table locals"><th>Nazwa</th><th>Adress</th><th>Oceny</th></table>'
+    );
+    $.each(localsToShow, (i, l) => {
+      $('.locals').append(
+        '<tr><td>' +
+          l.name +
+          '</td><td>' +
+          l.location.adress +
+          '</td><td><div class="stars1"><select class="stars" ><option value="1">1</option> <option value="2">2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option></select></div></td></tr>'
+      );
+      makingStars(l.rate);
+      $('.stars1').on('click', function() {
+        $('#myModal').modal('show');
+        createModal(l);
+      });
+    });
+  }
+}
+
+var id = '';
+
+$(document).ready(function() {
+  id = chceckIdCategory();
+
+  showCategory();
+});
+
+$(window).on('hashchange', function(e) {
+  id = chceckIdCategory();
+  showCategory();
+});
+
+function createModal(local) {
+  $('.ratesModalTitle').html('Oceny dla lokalu ' + local.name);
+  $('.ratesModalBody').html(
+    '<table class="table rate"><th>Uzytkownik</th><th>Ocena</th><th>Data</th><th>tres</th></table>'
+  );
+  $.each(rates, (i, r) => {
+    var date = r.created_at.split('-').reverse();
+    var now = moment().format('YYYY-MM-DD').split('-');
+    var diffDays = moment(now).diff(date, 'days');
+    var diffMonths = moment(now).diff(date, 'month');
+    var diffYears = moment(now).diff(date, 'year');
+    var message = '';
+    if (diffDays == 1) {
```

Comment:
moment.js sam udostępnia metodę zwracającą relatywny czas, możliwe że z polskim tlumaczeniem. obczaj 
https://momentjs.com/docs/#/displaying/fromnow/
https://momentjs.com/docs/#/customization/relative-time/
https://stackoverflow.com/questions/38367038/format-relative-time-in-momentjs